doc-files/release-notes-24.md

```diff
@@ -0,0 +1,223 @@
+
+For example:
+
+```
+java --enable-native-access=javafx.graphics,javafx.media,javafx.web
+```
+
+See [JDK-8347744](https://bugs.openjdk.org/browse/JDK-8347744) for more information.
+
+### The `jdk.jsobject` Module is Now Included with JavaFX
+
+The `jdk.jsobject` module, which is used by JavaFX WebView applications, is now included with JavaFX, replacing the JDK module of the same name. The `jdk.jsobject` module is deprecated as of JDK 24, and will be removed in a future release of the JDK.
+
+To facilitate the transition, `jdk.jsobject` is now an upgradable module in the JDK. This means that the version of `jdk.jsobject` delivered with JavaFX can be used in place of the one in the JDK to avoid the compiler warning. This can be done as follows:
+
+#### Applications using the SDK
+
+When running with the JavaFX SDK, use the `--upgrade-module-path` argument. For example:
+
+```
+javac --upgrade-module-path=/path/to/javafx-sdk-24/lib
+java --upgrade-module-path=/path/to/javafx-sdk-24/lib
+```
+
+NOTE: The above will fail if you run your application with JDK 23 or earlier. JDK 24 is recommended when running JavaFX 24, but if you choose to run JavaFX 24 with an earlier JDK, use the `--module-path` option instead.
+
+#### Applications using `jlink` to create a custom Java runtime image:
+
+When creating your custom Java runtime image, put the JavaFX jmods on the module path ahead of the JDK jmods. For example:
+
+```
+jlink --output jdk-with-javafx \
+    --module-path /path/to/javafx-jmods-24:/path/to/jdk-24/jmods \
+    --add-modules ALL-MODULE-PATH
+```
+
+NOTE: The above will fail if you create a custom image using JDK 23 or earlier. JDK 24 is recommended with JavaFX 24, but if you choose to run JavaFX 24 with an earlier JDK, put the JDK jmods ahead of the JavaFX jmods on the module path (that is, reverse the order of `javafx-jmods-24` and `jdk-24/jmods`).
+
+See [JDK-8337280](https://bugs.openjdk.org/browse/JDK-8337280) for more information.
+
+### Pluggable Image Loading via `javax.imageio`
+
+JavaFX 24 supports the Java Image I/O API, allowing applications to use third-party image loaders in addition to the built-in image loaders. This includes the ability to use variable-density image loaders for formats like SVG. When an image is loaded using a variable-density image loader, JavaFX rasterizes the image with the screen's DPI scaling.
+
+Applications that want to use this feature can use existing open-source Image I/O extension libraries, or register a custom Image I/O service provider instance with the `IIORegistry` class. Refer to the Java [Image I/O documentation](https://docs.oracle.com/en/java/javase/23/docs/api/java.desktop/javax/imageio/package-summary.html) for more information.
```

Comment:
NOTE to reviewers: I will change this to 24 with the next commit, but since it will be transiently broken, I wanted to give reviewers a chance to see the page with working links.

Reply:
Actually, I'll leave one as "23", since all of our API docs also point to 23 (since that's the boot JDK we use).